### PR TITLE
Added ESNext examples to format API tutorial (page 1)

### PR DESCRIPTION
**Pull request by bfintal**
Added Format API ESNext examples

### CODE REVIEW

File: docs/designers-developers/developers/tutorials/format-api/1-register-format.md [hunk 1]
```diff
@@ -38,6 +40,18 @@ Then add a new file named `my-custom-format.js` with the following contents:
...
+const { registerFormatType } = wp.richText;
+
+registerFormatType(
+	'my-custom-format/sample-output', {
+		title: 'Sample output',
+		tagName: 'samp',
+		className: null,
+	}
+);
+```
```
Comment (mkaz) [inline docs/designers-developers/developers/tutorials/format-api/1-register-format.md]: Looks like it is missing the closing `{% end %}` tag.

**Review by mkaz — CHANGES_REQUESTED** [review]

File: docs/designers-developers/developers/tutorials/format-api/1-register-format.md (hunk 1 above)
Comment (bfintal) [reply docs/designers-developers/developers/tutorials/format-api/1-register-format.md]: Oops sorry

**Comment by bfintal** [comment]
@mkaz I added in the end tag. Thanks!

**Review by mkaz — APPROVED** [review]
Thanks for the updates :+1:

**Pull request merged**